src/Reset.ts

```diff
@@ -900,6 +925,7 @@ export const singularity = async (): Promise<void> => {
     toggleSubTab(9, 0); // set 'corruption main'
     toggleSubTab(-1, 0); // set 'statistics main'
 
+    hold.history.singularity = player.history.singularity;
```

Comment:
?

Reply:
need to preserve the sing history over sing otherwise it will reset on sing and you will never see it